React/CoreModules/RCTRedBox.mm

```diff
@@ -968,6 +969,10 @@ - (void)showErrorMessage:(NSString *)message
 
     RCTErrorInfo *errorInfo = [[RCTErrorInfo alloc] initWithErrorMessage:message stack:stack];
     errorInfo = [self _customizeError:errorInfo];
+    
+    // TODO:(macOS GH#774) Explicitly log the error message to the Apple Console so production logs can be retrieved
+    os_log_error(OS_LOG_DEFAULT, "React-Native RedBox Error Message:\n%@ \nReact-Native RedBox Stack:\n%@", errorInfo.errorMessage, errorInfo.stack);
```

Comment:
this entire class is only for non production so its ok not to have any other build stage #ifdefs ?

Reply:
Yea I noticed this too. I moved the error to a location that should hit production